brewbean/client
# Update queries

## Changelog

- Changed queries as a result of https://github.com/brewbean/graphql/pull/5. Need to move to `stage` together as well.
- Deleted `BrewTrak` as it is a deprecated name; now called `BrewLog`
- Cache is changed; tested for recipes loosely, but not for beans (saw some things that could use changes)
- Closes https://github.com/brewbean/graphql/issues/4 with https://github.com/brewbean/graphql/pull/5

## How to test
- checkout `feat/update-queries` on `/client`
- checkout `feat/normalize-table-name` on `/graphql`
- run `hasura metadata apply --envfile .env.development` in `/graphql`
- run `hasura migration apply --envfile .env.development` in `/graphql`
- run `docker-compose up -d` from shared folder (ex. `/brewbean`)  - see [brewbean/docker/README.md](https://github.com/brewbean/docker/blob/main/README.md) for more details
- test on `localhost:3000` in browser
- run `docker logs --follow {container_id}` to see `/client` output
- run `docker-compose restart client` if client code freezes from changes
- make sure to run `docker-compose down` whenever you finish working
